src/tests/class_tests/openms/source/ProteaseDigestion_test.cpp

```diff
@@ -72,20 +72,6 @@ START_SECTION(([EXTRA] ProteaseDigestion(const ProteaseDigestion& rhs)))
     TEST_EQUAL(pd.getSpecificity(), pd2.getSpecificity());
 END_SECTION
 
-START_SECTION(([EXTRA] ProteaseDigestion& operator=(const ProteaseDigestion& rhs)))
```

Comment:
what happened to this one?

Reply:
this operator was never explicitly implemented or used.
And now with unique_ptr it is implicitly deleted.

Reply:
hmm maybe it is needed for pyOpenMS.